src/components/Spinner/README.md

```diff
@@ -55,6 +55,16 @@ Smaller than the default spinner.
 
 ---
 
+## Accessibility
+
+<!-- content-for: web -->
+
+SVGs after often displayed inconsistently. The `Spinner` components accessibility is highly contextual. When the parent component is focusable you'll need to set the `hasFocusableParent` prop for the appropriate `role` attribute to be applied.
```

Comment:
```suggestion
SVGs after often displayed inconsistently. The `Spinner` components accessibility is highly contextual. When the parent component is focusable you’ll need to set the `hasFocusableParent` prop for the appropriate `role` attribute to be applied.
```

Reply:
@kaelig I think this is a typo.

@AndrewMusgrave What about something like:

```
SVGs are often conveyed inconsistently to assistive technologies. The `Spinner` component's accessibility is also highly contextual. When the parent component is focusable, you'll need to set the `hasFocusableParent` prop for the appropriate `role` attribute to be applied.
```

We can also include this in an accessibility section in the doc, to also provide some context for `role="status"`.

Reply:
Yup it was just a typo 😄